ouroboros-consensus/src/Ouroboros/Consensus/Protocol/PBFT.hs

```diff
@@ -27,48 +32,39 @@ import           Codec.Serialise (Serialise (..))
 import qualified Codec.Serialise.Decoding as Dec
 import qualified Codec.Serialise.Encoding as Enc
 import           Control.Monad.Except
-import           Data.Map.Strict (Map)
-import qualified Data.Map.Strict as Map
+import           Data.Bimap (Bimap)
+import qualified Data.Bimap as Bimap
+import           Data.Reflection (Given (..))
 import           Data.Sequence (Seq)
 import qualified Data.Sequence as Seq
-import           Data.Tuple (swap)
 import           Data.Typeable (Typeable)
 import           Data.Word (Word64)
 import           GHC.Generics (Generic)
 
+import qualified Cardano.Chain.Common as CC.Common
+import qualified Cardano.Chain.Genesis as CC.Genesis
+import           Cardano.Crypto (ProtocolMagicId)
+
 import           Ouroboros.Network.Block
 
+import           Ouroboros.Consensus.Crypto.DSIGN.Cardano
 import           Ouroboros.Consensus.Crypto.DSIGN.Class
-import           Ouroboros.Consensus.Crypto.DSIGN.Ed448 (Ed448DSIGN)
 import           Ouroboros.Consensus.Crypto.DSIGN.Mock (MockDSIGN)
 import           Ouroboros.Consensus.Node (NodeId (..))
 import           Ouroboros.Consensus.Protocol.Abstract
 import           Ouroboros.Consensus.Protocol.Test
 import           Ouroboros.Consensus.Util.Condense
 
--- | Invert a map which we assert to be a bijection.
---   If this map is not a bijection, the behaviour is not guaranteed.
---
---   Examples:
---
---   >>> invertBijection (Map.fromList [('a', 1 :: Int), ('b', 2), ('c', 3)])
---   fromList [(1,'a'),(2,'b'),(3,'c')]
-invertBijection
-  :: Ord v
-  => Map k v
-  -> Map v k
-invertBijection
-  = Map.fromListWith const
-  . fmap swap
-  . Map.toList
-
 data PBftLedgerView c = PBftLedgerView
   -- TODO Once we have the window and threshold in the protocol parameters, we
   -- will use them here and remove the parameters from 'PBftParams' below.
```

Comment:
Create a ticket (here and in the ledger repo).

Reply:
Though it could be argued it actually _belongs_ here, since this is where we implement PBFT. Ask @ruhatch or @dnadales  perhaps.

Reply:
I'm not sure - I believe our handling of params so far is that they're all in the Ledger, even if they're parameters that affect the consensus protocol, but that could definitely change.

Reply:
These are not updatable paramaters. They are fixed for any particular instance of PBFT. So this can either be handled in the same way as global constants like K.

Reply:
Ok, then we shall leave them where they are :)